pacman/operations/placer_algorithms/one_to_one_placer.py

```diff
@@ -86,31 +86,34 @@ class OneToOnePlacer(RadialPlacer):
 
     def __call__(self, machine_graph, machine):
 
+        # Iterate over vertices and generate placements
+        progress = ProgressBar(
+            machine_graph.n_vertices + 3, "Placing graph vertices")
         # check that the algorithm can handle the constraints
         self._check_constraints(
             machine_graph.vertices,
             additional_placement_constraints={SameChipAsConstraint})
-
+        progress.update()
         # Get which vertices must be placed on the same chip as another vertex
         same_chip_vertex_groups = get_same_chip_vertex_groups(machine_graph)
 
+        progress.update()
         # Work out the vertices that should be on the same chip by one-to-one
         # connectivity
         one_to_one_groups = group_vertices(
             machine_graph.vertices, functools.partial(
-                _find_one_to_one_vertices, graph=machine_graph))
+                _find_one_to_one_vertices, graph=machine_graph), 18)
```

Comment:
magic number

Reply:
Agreed! 

I am going to try one alternative but I think that will be slower on machine with very large chains,